eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -10,11 +10,17 @@
       Tracked with https://github.com/dotnet/fsharp/issues/14765
     -->
     <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/8.0.2" />
+    <UsagePattern IdentityGlob="Microsoft.Build.Framework/17.11.0-preview-24178-03" />
+    <UsagePattern IdentityGlob="Microsoft.Build.Tasks.Core/17.11.0-preview-24178-03" />
+    <UsagePattern IdentityGlob="Microsoft.NET.StringToolsBuild.Framework/17.11.0-preview-24178-03" />
```

Comment:
That's probably not the right way of dealing with msbuld prebuilts? Would it need to be handled via darc @MichaelSimons? Can we for now add it to the xml as is and I will fix it after we merge, since we want to fix VS issue as soon as possible?

Reply:
got rid of them.